plugin/src/main/java/net/dzikoysk/funnyguilds/listener/region/BlockPlace.java

```diff
@@ -29,12 +30,23 @@ public BlockPlace(FunnyGuilds plugin) {
     @EventHandler(priority = EventPriority.LOWEST)
     public void onPlace(BlockPlaceEvent event) {
         PluginConfiguration config = plugin.getPluginConfiguration();
+        MessageConfiguration messages = plugin.getMessageConfiguration();
 
         Player player = event.getPlayer();
         Block block = event.getBlock();
         Material type = block.getType();
         Location blockLocation = block.getLocation();
 
+        if(type == Material.TNT) {
```

Comment:
literówka spacja przy `if`

Reply:
To w sumie w/e bo i tak będzie reformat całego kodu w #1654 razem z zasadami